src/test/java/com/amazon/sqs/javamessaging/ModifyWitTimeSecondsTest.java

```diff
@@ -0,0 +1,19 @@
+package com.amazon.sqs.javamessaging;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ModifyWaitTimeSecondsTest {
```

Comment:
Can you fix the typo in the java file name please?

Reply:
Fixed.